airflow/jobs/scheduler_job_runner.py

```diff
@@ -1192,7 +1192,7 @@ def _create_dag_runs(self, dag_models: Collection[DagModel], session: Session) -
                 )
                 active_runs_of_dags[dag.dag_id] += 1
             if self._should_update_dag_next_dagruns(
-                dag, dag_model, active_runs_of_dags[dag.dag_id], session=session
+                dag, dag_model, None, active_runs_of_dags[dag.dag_id], session=session
```

Comment:
```suggestion
                dag,
                dag_model,
                last_dag_run=None,
                total_active_runs=active_runs_of_dags[dag.dag_id],
                session=session,
```

this would improve readability dignificantly. It may also be a good idea to make those arguments keyword-only.

Reply:
ok, I turned last_dag_run and total_active_runs into keyword-only arguments